src/schemata/definitions/workflow_template/action_objects/getResources.yml

```
@@ -0,0 +1,46 @@
+---
+type: object
+title: Query Resources
+description: |
+  An action to make queries on the API/V2 resources 
+propertyNames:
+  enum:
+    - get_resources
+    - GetResources
+    - GET_RESOURCES
+additionalProperties:
+  type: object
+  required:
+    - tag
+  properties:
+    tag:
+      $ref: '#/definitions/resourceTag'
+    filter:
+      type: object
+      propertyNames:
+        $ref: '#/definitions/resourceTag'
+    page: 
+      type: object
+      propertyNames:
+        $ref: '#/definitions/resourceTag'
+    sort: 
+      type: object
+      propertyNames:
+        $ref: '#/definitions/resourceTag'
+    blocking:
+      type: boolean
+      description: |
+        Defines if the query action will block the subsequent steps
+      default: false
+
+example:
```

```suggestion
examples:
```
You need to run `yarn build`, which will generate the files that we eventually consume when adding this repo as a dep.

CI should then fail without this change, because `example` is not a valid JSON key.